README.md

```diff
@@ -119,7 +119,7 @@ USAGE
 * [`dvc features`](docs/features.md) - Create, view, or modify Features with the Management API.
 * [`dvc generate`](docs/generate.md) - Generate Devcycle related files.
 * [`dvc help`](docs/help.md) - Display help for dvc.
-* [`dvc identity`](docs/identity.md) - Update your DevCycle Identity.
+* [`dvc identity`](docs/identity.md)
```

Comment:
was this supposed to have another description in place of the update one?

Reply:
nice catch 👀 how's this for a description?
<img width="642" alt="image" src="https://github.com/DevCycleHQ/cli/assets/98763537/cc9d1f61-7abd-49eb-9052-cb4162aa736a">